tests/Microsoft.Data.Analysis.Tests/ArrowIntegrationTests.cs

```diff
@@ -49,14 +50,26 @@ public void TestArrowIntegration()
                 .Append("UByteColumn", false, new UInt8Array.Builder().AppendRange(Enumerable.Repeat((byte)1, 10)).Build())
                 .Build();
 
-            DataFrame df = DataFrame.FromArrowRecordBatch(originalBatch);
+            ArrowBuffer.BitmapBuilder nullBitmapBufferBuilder = new ArrowBuffer.BitmapBuilder();
```

Comment:
Can you make a new test for this scenario instead? That way we ensure the original test works as it previously did.

Reply:
Done.